.github/workflows/test.yaml

```diff
@@ -0,0 +1,244 @@
+name: test
+on: [push]
+env:
+  BUILDDIR: "builddir"
+  SRCDIR: "srcdir"
+  COVDIR: "coverage"
+  INSTALLDIR: "installdir"
+  PACKAGES: "gettext-devel git libtool make opensc openssl valgrind meson ninja-build bash-completion"
+jobs:
+  autotools:
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:latest
+    env:
+      BUILD_OPTS: --enable-coverage
+      COVERAGE: yes
+      EXTRA_PKGS: lcov python-pip
+    steps:
+      # Install packages
```

Comment:
I guess we can probably make use of the [GH container registry](https://docs.github.com/en/free-pro-team@latest/packages/getting-started-with-github-container-registry/about-github-container-registry) to avoid installing packages on each CI run. If you are willing to try that, I can give you access to the project; let me know.

Reply:
That would be nice to try!